deploy/ft/ft-create.js

```diff
@@ -13,6 +14,13 @@ async function start() {
 	let near = await initiateNearConnection(NETWORK_ID);
 	const fundingAccount = await near.account(FUNDING_ACCOUNT_ID);
 
+	//get amount to transfer and see if owner has enough balance to fund drop
+	let amountToTransfer = new BN(FT_DATA.balancePerUse).mul(new BN(NUM_KEYS * DROP_CONFIG.usesPerKey))
```

Comment:
```suggestion
	let amountToTransfer = new BN(FT_DATA.balancePerUse).mul(new BN(NUM_KEYS * DROP_CONFIG.usesPerKey)).toString()
```

Reply:
I was using this to compare to the amount of FTs the user was trying to transfer (i assumed `new BN(FT_DATA.balancePerUse).mul(new BN(NUM_KEYS * DROP_CONFIG.usesPerKey)) ` returns a number). 

later in the script, when passing amountToTransfer into ft_tranfer_call, i pass it in with .toString()